### PR TITLE
Add v1 of env-tests

### LINKED ISSUE #5498: Stakeholders can run a test suite after contract upgrade release
Ideally we are reusing tests from both 
- e2e tests
- protocol tests
- ongoing environment tests
- release tests
- release CI tests

### PR DESCRIPTION
**Pull request by nambrot**
### Description

We have a fair number of instances where we run "tests" against an environment, examples include:

- protocol tests
- e2e tests
- attestation/odis bots

Additionally, we want to get to a point where we can run these tests against release candidates in CI as in #5473. Therefore, I attempted to find a structure that allows us to reuse testing across these environments (local ganache, CI ganache, local geth cluster, remote testnets). I tried to implement a v1 of that by having importable jest test cases that can hopefully be flexibly enough configured. I.e. one can either just run them as yarn scripts in the command line or import them as test cases (i.e. protocol or e2e tests)

To address #5498, one can just run them via `yarn baklava-test` in the new `env-tests` package.

### Other changes

- Have `generateKeys` also return the address of the key

### Tested

- Run against baklava

### Related issues

- Fixes #5498

### CODE REVIEW

File: packages/env-tests/package.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
     "build": "tsc -b .",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "test": "jest --runInBand",
-    "baklava-test": "CELO_ENV=baklava jest --runInBand",
-    "alfajores-test": "CELO_ENV=alfajores jest --runInBand",
-    "mainnet-test": "CELO_ENV=rc1 jest --runInBand"
+    "baklava-test": "CELO_ENV=baklava CELO_PROVIDER=https://baklava-forno.celo-testnet.org jest --runInBand",
+    "alfajores-test": "CELO_ENV=alfajores CELO_PROVIDER=https://alfajores-forno.celo-testnet.org jest --runInBand",
+    "mainnet-test": "CELO_ENV=rc1 CELO_PROVIDER=https://rc1-forno.celo-testnet.org jest --runInBand"
```
Comment (AlexBHarley) [inline packages/env-tests/package.json]:
```suggestion
    "mainnet-test": "CELO_ENV=rc1 CELO_PROVIDER=https://forno.celo.org jest --runInBand"
```

File: packages/env-tests/src/scaffold.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  Root,
+  TransferFrom,
+  TransferTo,
+  Exchange,
+}
+
+export const ONE = new BigNumber('1000000000000000000')
+
+export async function clearAllFundsToRoot(kit: ContractKit, mnemonic: string) {
+  Array.from
```
Comment (AlexBHarley) [inline packages/env-tests/src/scaffold.ts]:
```suggestion
```

**Review by AlexBHarley — COMMENTED** [review]
Cool! I also wonder how much benefit there is to tests like this running once a day/hour on the various networks.

**Comment by nambrot** [comment]
@AlexBHarley That's the goal!

**Review by AlexBHarley — APPROVED** [review]

**Pull request merged**